teps/README.md

```diff
@@ -250,3 +250,4 @@ This is the complete list of Tekton teps:
 |[TEP-0107](0107-propagating-parameters.md) | Propagating Parameters | implemented | 2022-05-26 |
 |[TEP-0108](0108-mapping-workspaces.md) | Mapping Workspaces | implemented | 2022-05-26 |
 |[TEP-0110](0110-decouple-catalog-organization-and-reference.md) | Decouple Catalog Organization and Resource Reference | implementable | 2022-06-06 |
+|[TEP-0111](0111-propagating-workspaces.md) | Propagating Workspaces | proposal | 2022-06-03 |
```

Comment:
```suggestion
|[TEP-0111](0111-propagating-workspaces.md) | Propagating Workspaces | implementable | 2022-06-03 |
```

Reply:
this is not yet fixed, that's why linting is failing

Reply:
Oh it was in the README. Thanks